plenum/test/bls/test_bls_not_depend_on_node_reg.py

```diff
@@ -0,0 +1,72 @@
+from plenum.test.pool_transactions.helper import demote_node
+from plenum.test.test_node import TestNode, checkNodesConnected, ensureElectionsDone, ensure_node_disconnected
+from plenum.test.helper import sdk_send_batches_of_random_and_check
+
+from plenum.common.config_helper import PNodeConfigHelper
+from plenum.common.types import f
+
+nodeCount = 7
```

Comment:
I think we need to make nodeCount = 4 to make sue that we can order after demotion and restart

Reply:
ok